app/serializers/api/internal/v1/hearing_summary_serializer.rb

```diff
@@ -18,6 +19,8 @@ class HearingSummarySerializer
             name: hearing_summary.court_centre.name,
           }
         end
+
+        has_many :defence_counsels
       end
```

Comment:
Can you update the request specs so the swagger UI mentions that we can include hearing_summaries.defence_counsel in the api call to the GET prosecution cases endpoint?